direnv/.envrc

```
@@ -0,0 +1 @@
+use flake
```

```suggestion
if has nix_direnv_version; then
  use flake
fi
```

this is very annoying for anyone who uses direnv but not nix(-direnv)